vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextArea.java

```diff
@@ -220,6 +220,16 @@ public void setMinLength(int minLength) {
         getValidationSupport().setMinLength(minLength);
     }
 
+    /**
+     * Scrolls the textarea to the bottom. Requires the textarea to be of fixed
+     * height, so it can have a scrollbar.
+     *
+     */
+    public void scrollToBottom() {
+        getElement().executeJs(
+                "(() => this.shadowRoot.querySelector(\"vaadin-input-container\").scrollTop = this.shadowRoot.querySelector(\"vaadin-input-container\").scrollHeight)()");
```

Comment:
Dear @AbdullahGheith, we should probably rename the API to `scrollToEnd` to align with other components.

We would need an API within the web component version of `TextArea` which could be found here: https://github.com/vaadin/web-components/tree/main/packages/text-area. It is needed to avoid executing custom JS accessing private API from the Flow API, but instead directly use web component's API.

We could also consider introducing `scrollToStart` at the same time.

Could you please confirm if you would like to proceed with suggested changes?

Reply:
Thanks. I will implement the changes.

Reply:
@yuriy-fix I've implemented the changes

Reply:
@AbdullahGheith Hey! I've merged the web component part and will return to this PR once the web-components are released.